package.json

```diff
@@ -22,8 +22,8 @@
   "main": "src/index.js",
   "scripts": {
     "test": "jest",
-    "lint": "eslint .",
-    "lint:fix": "eslint . --fix",
+    "lint": "tslint --project tsconfig.json",
```

Comment:
did you install tslint globally? Why not locally?

Reply:
It's installed as a dependency of our inaccurately named `eslint-config-twine repo`